frodo2-runtime-android/build.gradle

```diff
@@ -48,68 +25,73 @@ android {
   }
 
   compileOptions {
-    targetCompatibility JavaVersion.VERSION_1_7
-    sourceCompatibility JavaVersion.VERSION_1_7
-  }
-}
-
-android.libraryVariants.all { variant ->
-  LibraryPlugin plugin = project.plugins.getPlugin(LibraryPlugin)
-
-  //Compile aspects
-  JavaCompile javaCompile = variant.javaCompile
-  javaCompile.doLast {
-    String[] args = ["-showWeaveInfo",
-                     "-1.5",
-                     "-inpath", javaCompile.destinationDir.toString(),
-                     "-aspectpath", javaCompile.classpath.asPath,
-                     "-d", javaCompile.destinationDir.toString(),
-                     "-classpath", javaCompile.classpath.asPath,
-                     "-bootclasspath", plugin.project.android.bootClasspath.join(File.pathSeparator)]
-
-    final MessageHandler handler = new MessageHandler(true);
-    final def log = project.logger
-    new Main().run(args, handler)
-    for (IMessage message : handler.getMessages(null, true)) {
-      switch (message.getKind()) {
-        case IMessage.ABORT:
-        case IMessage.ERROR:
-        case IMessage.FAIL:
-          log.error message.message, message.thrown
-          break;
-        case IMessage.INFO:
-          log.info message.message, message.thrown
-          break;
-        case IMessage.DEBUG:
-          log.debug message.message, message.thrown
-          break;
-      }
-    }
-  }
-
-  //Change aar file name: add version name only for debug version (development)
-  if (variant.name == 'debug') {
-    variant.outputs.each { output ->
-      if (output.outputFile != null && output.outputFile.name.endsWith('.aar')) {
-        def file = output.outputFile
-        def fileName = file.name.replace("${variant.buildType.name}.aar", "${project.version}.aar")
-        output.outputFile = new File(file.parent, fileName)
-      }
-    }
+    targetCompatibility JavaVersion.VERSION_1_8
+    sourceCompatibility JavaVersion.VERSION_1_8
   }
 }
 
+// TODO: Do we need this?
```

Comment:
we do need it. We need to invoke the aspectj weaver for all the variants and libraries in the android project to process annotations and inject debugging code.

Reply:
I'm not sure it does. Can we keep this commented out for now until we fix it?

Reply:
but why you commented this?